packs/classfeatures/ruffian.json

```
@@ -135,4 +136,4 @@
         }
     },
     "type": "feat"
-}
+}
```

```suggestion
}

```
Check your IDE's settings if it deleted this automatically.